chia/util/block_cache.py

```diff
@@ -74,6 +77,12 @@ async def get_block_records_at(self, heights: List[uint32]) -> List[BlockRecord]
     async def get_block_record_from_db(self, header_hash: bytes32) -> Optional[BlockRecord]:
         return self._block_records[header_hash]
 
+    async def prev_block_hash(self, header_hashes: List[bytes32]) -> List[bytes32]:
```

Comment:
supporting multiple lookups in a single call matters in the case of making database lookups.
I'll add a comment

Reply:
it seems like we don't document these in `blockchain_interface.py`, but just in `consensus/blockchain.py`. I would think it would make the most sense to add it there, to follow the existing practice.